pkg/asset/machines/master.go

```diff
@@ -109,6 +123,7 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 
 	pool := ic.ControlPlane
 	var err error
+	hostSettings := &baremetal.HostSettings{}
```

Comment:
again, make it specific to baremetal..

Reply:
also why is this not in bare-metal switch..

Reply:
This initializes an empty variable. Line 232 gives a populated version. By ensuring the variable always exists, we don't need multiple platform checks before using it.